### ISSUE TITLE
[suggestion] Add github topics to repo

### ISSUE THREAD

**Issue by protron** [davidrmiller/biosim4#11]
After seeing your excellent video I wanted to simulate more types of evolution, but I don't have much C++ experience, and I don't have Ubuntu installed, so instead of trying to see if I could make this work with WSL, I first tried to find if someone else made some other open source evolution simulator. And searching online that I found this github topic: [evolution-simulator](https://github.com/topics/evolution-simulator). And it seems like your repo already has more stars than any other in there (so it seems like good free advertisement).

My suggestion is to add some topics to your repository to increase the chances to get more contributions (and views in your video).
It's really easy, here is github explanation for how it works: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics

**Comment by geoffarnold** [comment]
As I posted in Issue#1, getting this to work on WSL is very easy. Only real complexity is updating the version of Ubuntu.

**Comment by venzen** [comment]
I agree with the proposal to add topics for easily classifying issues, e.g. "urgent", "wishlist", etc.
